### DISCUSSION TITLE
App Cleaner Strucks At DCIM/Screenshots 64% 5 times in a row

### DISCUSSION THREAD

**Discussion by tathastu871** [d4rken-org/sdmaid-se#89] [General]
![Screenshot_2023-04-02_195908](https://user-images.githubusercontent.com/65007212/229359381-081a7808-0187-4b0d-860d-055a1f177bde.jpg)

**Comment by d4rken** [comment]
Many files in that folder? Can you provide a debug log?

**Comment by simsimient1** [comment]
That folder remove? Can you bug fix log?

**Comment by DeXbot2022** [comment]
Lol
